tests/Paket.Tests/Resolver/DependencyGroupsAndRestrictions.fs

```diff
@@ -104,7 +104,7 @@ let graph2 =
 </package>"""
   ]
   
-[<Test>]
+[<Test>][<Ignore("Currently failing")>]
 let ``should prefer netstandard1.3 dependency over all frameworks``() = 
```

Comment:
@forki Can you please take a look at this test and remove it if you don't like this behavior. I feel like this is how it should work. But maybe its not worth the effort right now as everything else seems to work now :+1: